pkgs/tools/security/web-eid-app/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, cmake
+, gtest
+, pcsclite
+, pkg-config
+, qmake
+, qttranslations
+}:
+
+mkDerivation rec {
+  pname = "web-eid-app";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "web-eid";
+    repo = "web-eid-app";
+    rev = "v${version}";
+    sha256 = "sha256-TOzOcPY4m7OdCfaAXyc/joIHe2O2YbyDrXiNytPMKSk=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    gtest
+    pcsclite
+    qttranslations
+  ];
+
+  meta = with lib; {
+    longDescription = ''
```

Comment:
Could you add a (short) description as well? search.nixos.org doesn't seem to use the longDescription if a description is absent. e.g. libao/libaosd on https://search.nixos.org/packages?channel=22.11&from=0&size=50&sort=relevance&type=packages&query=libao

Reply:
Updated.